Cargo.toml

```diff
@@ -48,4 +48,8 @@ path = "chalk-engine"
 version = "0.1.0"
 path = "chalk-rust-ir"
 
+[dependencies.fold-derive]
```

Comment:
Maybe name it `chalk-macros`?

Reply:
There's already a `chalk-macros` crate, and we can't add these to that crate:
```
`proc-macro` crate types cannot export any items other than functions tagged with `#[proc_macro_derive]` currently
```
Maybe `chalk-derive` instead?